hugo/Caddyfile

```diff
@@ -0,0 +1,11 @@
+blog.mysite.tld:443 {
+        root /var/www/blog/public
```

Comment:
Here's another assumption that is not explained in the readme; remove it or explain it I think.